apps/api/prisma/schema.prisma

```diff
@@ -349,11 +349,9 @@ model DocumentVersion {
   stage                  String?
```

Comment:
DocumentVersion should look something like this

`model DocumentVersion {
  documentGuid             String 
  version                Int       @default(1)

...

  @@id([documentGuid, version])
  @@index([filter1], map: "filter1")
}`